lib/scala/bench-processor/src/main/java/org/enso/benchmarks/processor/BenchProcessor.java

```diff
@@ -438,6 +438,8 @@ private static char peekLastChar(StringBuilder sb) {
   }
 
   private void failWithMessage(String msg) {
+    // Better have duplicated error message than none at all
+    System.out.println("[org.enso.benchmarks.processor.BenchProcessor]: ERROR: " + msg);
```

Comment:
```suggestion
    System.err.println("[org.enso.benchmarks.processor.BenchProcessor]: ERROR: " + msg);
```

Reply:
I have already tried that, but the output from std err is just lost somewhere. Printing directly to `System.out` ensures that it is at least visible in the sbt shell.